test/components/dragBoxLayerTests.ts

```diff
@@ -475,5 +475,138 @@ describe("Interactive Components", () => {
+        dbl.movable(true);
+        TestMethods.triggerFakeDragSequence(target,
+          { x: midPoint.x, y: midPoint.y },
+          { x: midPoint.x + dragDistance, y: midPoint.y }
+        );
+        var bounds = dbl.bounds();
+        assert.strictEqual(bounds.topLeft.x, initialBounds.topLeft.x + dragDistance, "left edge moved");
+        assert.strictEqual(bounds.bottomRight.x, initialBounds.bottomRight.x + dragDistance, "right edge moved");
+        assert.strictEqual(bounds.topLeft.y, initialBounds.topLeft.y, "top edge did not move");
+        assert.strictEqual(bounds.bottomRight.y, initialBounds.bottomRight.y, "bottom edge did not move");
+        svg.remove();
+      });
+
+      it("move up", () => {
+        dbl.movable(true);
+        TestMethods.triggerFakeDragSequence(target,
+          { x: midPoint.x, y: midPoint.y },
+          { x: midPoint.x, y: midPoint.y - dragDistance }
+        );
+        var bounds = dbl.bounds();
+        assert.strictEqual(bounds.topLeft.x, initialBounds.topLeft.x, "left edge did not move");
+        assert.strictEqual(bounds.bottomRight.x, initialBounds.bottomRight.x, "right edge did not move");
+        assert.strictEqual(bounds.topLeft.y, initialBounds.topLeft.y - dragDistance, "top edge moved");
+        assert.strictEqual(bounds.bottomRight.y, initialBounds.bottomRight.y - dragDistance, "bottom edge moved");
+        svg.remove();
+      });
+
+      it("move down", () => {
+        dbl.movable(true);
+        TestMethods.triggerFakeDragSequence(target,
+          { x: midPoint.x, y: midPoint.y },
+          { x: midPoint.x, y: midPoint.y + dragDistance }
+        );
+        var bounds = dbl.bounds();
+        assert.strictEqual(bounds.topLeft.x, initialBounds.topLeft.x, "left edge did not move");
+        assert.strictEqual(bounds.bottomRight.x, initialBounds.bottomRight.x, "right edge did not move");
+        assert.strictEqual(bounds.topLeft.y, initialBounds.topLeft.y + dragDistance, "top edge moved");
+        assert.strictEqual(bounds.bottomRight.y, initialBounds.bottomRight.y + dragDistance, "bottom edge moved");
+        svg.remove();
+      });
+
+      it("does not move if grabbed within detectionRadius() while resizable()", () => {
+        dbl.movable(true);
+        dbl.resizable(true);
+        TestMethods.triggerFakeDragSequence(target,
+          { x: initialBounds.bottomRight.x, y: midPoint.y },
+          { x: SVG_WIDTH, y: midPoint.y }
+        );
+        var bounds = dbl.bounds();
+        assert.strictEqual(bounds.topLeft.y, initialBounds.topLeft.y, "top edge was not moved");
+        assert.strictEqual(bounds.bottomRight.y, initialBounds.bottomRight.y, "bottom edge was not moved");
+        assert.strictEqual(bounds.topLeft.x, initialBounds.topLeft.x, "left edge was not moved");
+        assert.strictEqual(bounds.bottomRight.x, SVG_WIDTH, "right edge was repositioned");
+        svg.remove();
+      });
+
+      it("doesn't dismiss on no-op move", () => {
```

Comment:
Add test for it dismisses box when clicked outside of it no-op (basically same test but x: 0, y: 0 for both start drag and end drag)